tzatziki-spring-mongodb/src/main/java/com/decathlon/tzatziki/steps/SpringMongoSteps.java

```diff
@@ -46,6 +51,51 @@ public SpringMongoSteps(ObjectSteps objects, SpringSteps spring) {
 
     @Before
     public void before() {
+        if (autoclean) {
```

Comment:
I think you can write a test for this part:
Create a custom step to run this 3 lines, write a test which insert some data, execute the custom step for deletion and check the database is empty.

Reply:
Done.